src/test/eo/org/eolang/collections/list-tests.eo

```diff
@@ -588,3 +588,16 @@
       lt. > @
         ^.at 0
         x.at 0
+
+# This code should prints "0 1"
+[] > reduced-printing-test
+  reduced. > @
+    list
+      * 0 1
+    TRUE
```

Comment:
What does this object `TRUE` mean? As I understand (https://github.com/objectionary/eo-collections/blob/master/src/main/eo/org/eolang/collections/list.eo#L70-L77), it is an initial value for `reduced`, but array is `[0, 1]`. Correct me if I'm wrong here.